src/useq/_grid.py

```diff
@@ -189,35 +164,41 @@ def _step_size(self, fov_width: float, fov_height: float) -> Tuple[float, float]
         return dx, dy
 
 
-class GridFromCorners(_GridPlan):
+class GridFromEdges(_GridPlan):
     """Define grid positions from two corners.
 
     Attributes
     ----------
-    corner1 : Coordinate
-        First bounding coordinate (e.g. "top left"). The position is considered
-        to be in the center of the image.
-    corner2 : Coordinate
-        Second bounding coordinate (e.g. "bottom right"). The position is considered
-        to be in the center of the image.
+    top : float
+        y value of the top left bounding coordinate
+    left : float
+        x value of the top left bounding coordinate
+    bottom : float
+        y value of the bottom right bounding coordinate
+    right : float
+        x value of the bottom right bounding coordinate
+
+    Values are considered to be in the center of the image.
```

Comment:
not sure what this means... i would assume the values were at the the edges of the image

Reply:
> not sure what this means... i would assume the values were at the the edges of the image

it means that the value is considered to be a position at the center of the FOV and not for example the top left...Then in the general bounding box the top value will indeed define the top edge of the box. Does it make any sense?

Reply:
ahhh, I think I see what you mean.  you're saying that when someone has a live view on, and they "mark the top" it's whatever stage position that's at... not the stage position - 1/2 FOV?

If so, I think that needs a fuller explanation, perhaps an example, etc...

Reply:
and actually... and argument could be made that useq schema should _not_ care about that. And that that math should be done before passing to useq (or that we have a special constructor for that or something).  a bounding box is a simple, clear concept... it's a nice model.  a bounding box taking into consideration some as-of-yet unknown field of view is not so much a good model.

Reply:
(sorry, one more thought):  just say that the position is the stage position (and not relative to an image) rather than saying it's at the center of the FOV.

Reply:
what about something like this?

```python
class GridFromEdges(_GridPlan):
    """Yield absolute stage positions to cover a bounded area...
    
    ...defined by setting its top, left, bottom and right
    edges stage coordinates.

    Attributes
    ----------
    top : float
        stage position of the top bounding area
    left : float
        stage position of the left bounding area
    bottom : float
        stage position of the bottom bounding area
    right : float
        stage position of the right bounding area
    """
```

Reply:
Sure :) but make them "... of the bounding area".  Currently reads like there are four areas

Reply:
Or just make them "top bounds", "left bounds", etc... and then say "all postitions in stage coordinates"